src/consumer/component/sources/dist.js

```diff
@@ -51,7 +50,7 @@ export default class Dist extends Vinyl {
 
   buildSrcWithSourceMapAnnotation(fileName: string) {
     return this.mappings ?
-    `${this.contents.toString()}\n\n//# sourceMappingURL=${fileName}${MAP_EXTENSION}` : this.contents.toString();
+    `${this.compiledContent.toString()}\n\n//# sourceMappingURL=${fileName}${MAP_EXTENSION}` : this.contents.toString();
```

Comment:
It doesn't make sense to me that one of the conditions return compiledContent and one content. but i might be wrong.
Check it.